tasks/overcloud_images.yml

```diff
@@ -0,0 +1,69 @@
+- block:
+    - name: create images destination
+      file:
+        path: /home/stack/images
+        state: directory
+        owner: stack
+        mode: "0755"
+
+    - name: extract images
+      unarchive:
+        remote_src: true
+        dest: /home/stack/images
+        owner: stack
+        src: "{{ item }}"
+      with_items:
```

Comment:
I wonder if we can start using `loop` now that Ansible 2.8 is our minimum required version.

Reply:
`loop` came in 2.5, so there's no specific need for 2.8 to use it.

Reply:
`with_*` is just a way of combining loop with a lookup. Is `loop` now the preferred syntax for this?

Reply:
I believe so:

https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#iterating-over-a-simple-list